### PR TITLE
Adds versioning in

### PR DESCRIPTION
**Pull request by stevenjack**
### Problem

Repo doesn't have versioning, it really should.

### Solution

* Add it.

### CODE REVIEW

**Comment by stevenjack** [comment]
[[FBI 1.0.0] Adds versioning](https://trello.com/c/rsfJFH34/964-fbi-1-0-0-adds-versioning)

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+PATH_BASE ?= "/go/src/github.com/vidsy"
+GO_BUILDER_IMAGE ?= "vidsyhq/go-builder"
+REPONAME ?= "fbintegration"
+SSH_KEY_NAME ?= "id_circleci_github"
+BRANCH = "master"
+VERSION = $(shell cat ./VERSION)
+
+DEFAULT: test
+
+install:
```
Comment (revett) [inline Makefile]: Change to `install-dependencies`

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	git pull origin ${BRANCH}
+	git tag ${VERSION}
+	git push origin ${BRANCH} --tags
+
+test:
+	@echo "No tests yet :("
+
+test_ci: test
+
+test_coverage:
```
Comment (revett) [inline Makefile]: Use `-`

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	@echo "=> New tag version: ${VERSION}"
+	git checkout ${BRANCH}
+	git pull origin ${BRANCH}
+	git tag ${VERSION}
+	git push origin ${BRANCH} --tags
+
+test:
+	@echo "No tests yet :("
+
+test_ci: test
```
Comment (revett) [inline Makefile]: Use `-`

**Pull request merged**